sbom.cdx.json

```diff
@@ -0,0 +1,43 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "version": 1,
+  "metadata": {
+    "authors": [
+      {
+        "name": "@VCS_SBOM_AUTHORS@"
```

Comment:
Who is supposed to substitute these keywords? With what tools?

In particular, needing an extra tool in the environment where we prepare release tarballs would require some work on our part.

Reply:
A good question, but no extra work required on your part. Those tokens are populated automatically using the [uswid tool](https://github.com/hughsie/python-uswid/blob/main/README.md#substituted-values) at build time -- and also whatever non-free build tools the firmware teams use. The tokens themselves are only standardized in uswid at the moment, but will soon move to the [Firmware Embedded SBOM Specification](https://lvfs.readthedocs.io/en/latest/sbom.html) where they'll be concreted into place. We've got quite a lot of support from other projects now -- e.g. see https://docs.google.com/spreadsheets/d/1gKEWLxdLubOfgS1cqqY2umEX0ohYEoEy-B1i59HNVqg/edit?usp=sharing -- and so I think the `@TOKENS@` are here to stay now.

Reply:
> no extra work required on your part

Currently you can get our releases either via a git tag or via a tarball. How could the `CVS_xxx` tokens get populated from a release tarball?

Reply:
Good question; I think all the firmware is currently built using git tags rather than tarballs, but I do appreciate the tarball case. In most cases for other projects the sbom isn't actually in the list of files that gets included in the release tarball (as like you said, the values are not auto-populated) -- but if you **do** want it in the tarball we can do something like https://github.com/fwupd/fwupd/blob/main/contrib/meson.build#L15 -- I'd need a bit of a hand with the CMake stuff if so please.

Reply:
Having a usable SBOM in the tarball would be nice. If you don't need it, though, it can wait until someone needs it. At the moment, we don't have bandwidth to investigate, and our build and release scripts are changing heavily, so if we can avoid changing those scripts for now, all the better.